Cesium3DTilesSelection/src/ITwinUtilities.cpp

```diff
@@ -0,0 +1,75 @@
+#include "ITwinUtilities.h"
+
+#include <CesiumAsync/IAssetResponse.h>
+#include <CesiumUtility/ErrorList.h>
+#include <CesiumUtility/JsonHelpers.h>
+
+#include <fmt/format.h>
+#include <rapidjson/document.h>
+
+#include <cstddef>
+#include <span>
+#include <string>
+
+namespace Cesium3DTilesSelection {
+void parseITwinErrorResponseIntoErrorList(
+    const CesiumAsync::IAssetResponse* pResponse,
+    CesiumUtility::ErrorList& errors) {
+  rapidjson::Document jsonResponse;
+  const std::span<const std::byte> data = pResponse->data();
```

Comment:
You should either check `pResponse` against nullptr, or pass it into this function by reference instead of pointer.